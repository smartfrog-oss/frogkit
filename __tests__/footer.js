import { mount } from 'vue-test-utils'
import Footer from '@/components/footer'

const navItems = [
  [
    { label: 'Häufig gestellte Fragen (FAQ)', url: 'https://support.smartfrog.com/hc/de' },
    { label: 'Support', url: 'https://www.smartfrog.com/de-de/support/' },
    { label: 'Kontakt', url: 'https://www.smartfrog.com/de-de/kontakt/' },
    { label: 'Impressum', url: 'https://www.smartfrog.com/de-de/impressum/' },
    { label: 'Bezugsquellen', url: 'https://www.smartfrog.com/de-de/bezugsquellen/' }
  ],
  [
    { label: 'Nutzungsbedingungen', url: 'https://www.smartfrog.com/de-de/nutzungsbedingungen/' },
    { label: 'Datenschutzrichtlinie', url: 'https://www.smartfrog.com/de-de/datenschutzrichtlinie/' },
    { label: 'Cookies', url: 'https://www.smartfrog.com/de-de/cookies/' },
    { label: 'Glossar', url: 'https://www.smartfrog.com/de-de/glossar' },
    { label: 'Versand', url: 'https://www.smartfrog.com/de-de/versand/' }
  ],
  [
    { label: 'Über uns', url: 'https://www.smartfrog.com/de-de/ueber-uns/' },
    { label: 'Jobs', url: 'https://www.smartfrog.com/de-de/jobs/' }
  ],
  [
    { label: 'Blog', url: 'https://blog.smartfrog.com/de' },
    { label: 'Presse', url: 'https://www.smartfrog.com/de-de/presse/' }
  ]
]
const appStoreItems = {
  headline: 'App downloaden:',
  buttons: [
    {
      name: 'apple',
      url: 'https://app.adjust.com/p39urb',
      label: 'Laden im'
    },
    {
      name: 'google',
      url: 'https://app.adjust.com/vti0vj',
      label: 'JETZT BEI'
    }
  ]
}
const socialItems = {
  headline: 'Smartfrog folgen:',
  icons: [
    {
      name: 'facebook',
      url: 'http://www.facebook.com/smartfrogDE',
      label: 'Folgen Sie auf Facebook'
    },
    {
      name: 'twitter',
      url: 'https://twitter.com/smartfrog_cam',
      label: 'Auf Twitter folgen'
    },
    {
      name: 'youtube',
      url: 'https://www.youtube.com/channel/UCt2tBYCxnfhBWraIuOcY2Ow',
      label: 'Folgen Sie auf YouTube'
    },
    {
      name: 'instagram',
      url: 'http://instagram.com/smartfrogcam/',
      label: 'Folgen Sie auf Instagram'
    }
  ]
}

const data = {
  navItems: navItems,
  appStoreItems: appStoreItems,
  socialItems: socialItems
}

describe('Footer component', () => {
  it('Should render component and match snapshot', () => {
    const wrapper = mount(Footer, { propsData: data })
    expect(wrapper.html()).toMatchSnapshot()
  })
})
