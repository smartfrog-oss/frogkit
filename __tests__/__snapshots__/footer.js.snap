// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component Should render component and match snapshot 1`] = `
<footer class="fk-footer">
  <grid>
    <row>
      <col xs12="" lg3="">
      <col xs12="" lg3="">
      <col xs12="" lg3="">
      <col xs12="" lg3="">
    </row>
  </grid>
</footer>
`;
